extensions/interactions/NumberWithUnits/directives/number-with-units-interaction.component.html

```diff
@@ -1,14 +1,21 @@
-<form autocomplete="off" ng-disabled="$ctrl.NumberWithUnitsForm.$invalid" ng-submit="$ctrl.submitAnswer($ctrl.answer)" role="form" class="form-horizontal">
+<form autocomplete="off" [attr.disabled]="!isValid ? true : false" role="form" class="form-horizontal">
```

Comment:
```suggestion
<form autocomplete="off" [disabled]="!isValid ? true : false" role="form" class="form-horizontal">
```

attr is not required here.

Reply:
Done